assets/main.scss

```
@@ -141,3 +141,16 @@ article table {
     @extend .bg-info-subtle;
   }
 }
+
+details:not([open]) summary i {
+  @extend .d-inline-block;
+  transform: rotate(-90deg); 
+}
+
+.keyword-list {
+  @extend .d-block
+}
+
+.form-check:has(#flexSwitchCheckDefault:checked) ~ .keyword-list{
```

Not sure the `.keyword-list` class is contributing much here. Could we do something like this instead?

```suggestion
.form-check:has(#flexSwitchCheckDefault:checked) ~ ul {
```

---

Also mind the space before `{`!

---

I need this class only because I need to apply functionality only for the first ul. Is it ok to have it?

---

Not sure I get it 100%, but could you use `:first-child` or something like that?